src/hooks/useAlleleSelection.js

```diff
@@ -66,7 +66,23 @@ export default function useAlleleSelection(tableProps) {
           for (const allele of validAlleles) {
             if (allele && allele.id && !seenIds.has(allele.id)) {
               seenIds.add(allele.id);
-              uniqueAlleles.push(allele);
+
+              // Compute correct category based on variant count
+              // The individual allele API doesn't return the computed category
+              // so we need to calculate it based on the variants array
```

Comment:
This should change in 8.3. The individual allele endpoint should have this field once it's transitioned to Postgres data.